resources/js/Components/Sticker.vue

```diff
@@ -0,0 +1,30 @@
+        </clipPath>
+    </svg>
+
+    <div class="group relative sticker-clip-path w-[220px] h-[220px] flex justify-center basis-0 overflow-hidden" :class="`bg-${color}` || {[`grayscale`] : !active}" >
```

Comment:
Even though `color` can be the empty string (and thus have a falsy value), the expression `bg-{color}` will always have a truthy value, making it the only value to ever be used.

One way of doing this could be like the following:

```suggestion
    <div class="group relative sticker-clip-path w-[220px] h-[220px] flex justify-center basis-0 overflow-hidden" :class="active ? `bg-${color}` || 'grayscale'" >
```

The syntax might not be right but you get the point. If I'm mistaken please correct me.

Reply:
As a bonus, we could look into the [clsx](https://www.npmjs.com/package/clsx) package on NPM to handle this kinds of things for us.

Reply:
Also, won't we run into problems for having the `class` attribute repeated? I genuinely don't know.

Reply:
I don't think that syntax would work, the goal is not to have an if statement where we check `active` and either apply the class `grayscale` or `bg-${color}`, but rather have the bind apply `bg-${color}` everytime and `grayscale` only when `!active` is true. I'm not sure if this is the behavior the design team intended but for a first draft should be good enough at showing if a sticker has been awarded or not.
Regarding the repeated `class` attribute, this is an intended use of Vue's [class/style bindings](https://vuejs.org/guide/essentials/class-and-style.html#binding-html-classes).

Reply:
The link you provided mentions binding `class` to Arrays, take a look at those examples since I believe that is what you wanted to achieve.